get_web_content.pl

```diff
@@ -76,6 +77,18 @@
         my $img_file = $image->{file};
         $image->{outfile} = "$TaskData/$img_file";
         my $got_image = 0;
+
+        if (exists $web->{add_date}) { # Ugh, custom code for magnetogram
```

Comment:
If this is just for the solar_flare_monitor, I might skip the special "add_date" key and just make this for the solar_flare_monitor explicitly (I think that would be $web_name == 'solar_flare_monitor' but haven't checked).